src/main/java/mods/railcraft/common/items/enchantment/EnchantmentBoost.java

```diff
@@ -0,0 +1,37 @@
+/*------------------------------------------------------------------------------
+ Copyright (c) CovertJaguar, 2011-2017
+ http://railcraft.info
+
+ This code is the property of CovertJaguar
+ and may only be used with explicit written
+ permission unless otherwise specified on the
+ license page at http://railcraft.info/wiki/info:license.
+ -----------------------------------------------------------------------------*/
+package mods.railcraft.common.items.enchantment;
+
+import net.minecraft.inventory.EntityEquipmentSlot;
+
+/**
+ * Boost enchantment.
+ */
+public class EnchantmentBoost extends EnchantmentToolRailcraft {
+
+    public EnchantmentBoost(Rarity rarity) {
+        super("boost", rarity, EntityEquipmentSlot.MAINHAND);
```

Comment:
Read more code! This has no effect as boosting is totally done in crowbar event handler.